components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -482,6 +484,10 @@
   margin: .5em .75% 0em !important;
   padding-right: .75%;
 }
+.non-moving {
+  margin-top: 75px !important;
```

Comment:
I originally had it temporarily to overwrite another !important but forgot to remove it. Good catch!